### PR TITLE
WIP: author name now on detail page

### PR DESCRIPTION
**Pull request by JordanDDisch**

### CODE REVIEW

**Comment by edisch** [comment]
I concur. :+1:

File: lib/load-node/index.js [hunk 1]
```diff
@@ -8,17 +8,26 @@ module.exports.loadNode = function loadNode(nodeRequestConfig, node) {
...
-                    }
-                    PromiseArray = [];
+                    };
                     fileRequestConfig.params.fid = data.field_image.file.id;
                     PromiseArray.push(getResource(fileRequestConfig));
                     fileRequestConfig.params.fid = data.field_youtube_video.file.id;
                     PromiseArray.push(getResource(fileRequestConfig));
+                    var userRequestConfig = {
+                        resource: 'user',
+                        params: {
```
Comment (rcaracaus) [inline lib/load-node/index.js]: Should we make the limit be 1 by default in the service?

File: lib/load-node/index.js [hunk 2]
```diff
@@ -8,17 +8,26 @@ module.exports.loadNode = function loadNode(nodeRequestConfig, node) {
...
+                            limit: 1
+                        }
+                    };
+                    userRequestConfig.params.uid = data.author.id;
```
Comment (rcaracaus) [inline lib/load-node/index.js]: we need to test separately for author and we might as well do youtube as well

**Comment by edisch** [comment]
You committed the bower_components folder. 126 files.

**Pull request closed without merging**